src/postgres-util/src/lib.rs

```diff
@@ -440,6 +440,8 @@ impl Config {
             config
                 .replication_mode(ReplicationMode::Logical)
                 .connect_timeout(Duration::from_secs(30))
+                .keepalives_interval(Duration::from_secs(60))
+                .tcp_user_timeout(Duration::from_secs(60))
```

Comment:
I'm not sure how hard it is to wire in to this point, but it could be nice for these to be dynamically configurable so we could roll out changes without needing a new build

Reply:
60s is probably a safe starting point, but with the variability of the networking stacks sitting between us and all possible sources, I wonder if we'll want to fiddle with this value.

Reply:
(this could also help the test case, where we could set the timeout very low and not have to wait 2 min)